src/decode/lzma.rs

```diff
@@ -1,6 +1,7 @@
 use crate::decode::lzbuffer::{LzBuffer, LzCircularBuffer};
 use crate::decode::rangecoder;
 use crate::decode::rangecoder::RangeDecoder;
```

Comment:
It'd make more sense to import BitTree directly and merge these two lines together.